tpgtools/api/eventarc/samples/copy.py

```diff
@@ -0,0 +1,24 @@
+import os
```

Comment:
Hmm- I'm not really sure what this is from / what it's for?

Reply:
Ohh yes I need to remove this file, this was a script I ran to copy samples from DCL into this directory, will delete this

Reply:
Delete the file